modules/govuk/manifests/apps/local_links_manager.pp

```diff
@@ -104,6 +104,7 @@
   $run_links_ga_export = false,
 ) {
   $app_name = 'local-links-manager'
+  $unicorn_worker_processes = 4
```

Comment:
would this make more sense in the hiera data instead of being declared inline (I know this is whats done for whitehall)

Reply:
shrug.  I just followed what whitehall does - but no idea really if that's the preferred / best way.

Reply:
Actually, after talking about it on Slack, it seems like using hieradata might be a better option.  I'll change it.